src/Sql/Sql/help/Remove-AzSqlDatabaseInstanceFailoverGroup.md

```diff
@@ -0,0 +1,199 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version: https://docs.microsoft.com/en-us/powershell/module/az.sql/remove-azsqldatabaseinstancefailovergroup
+schema: 2.0.0
+---
+
+# Remove-AzSqlDatabaseInstanceFailoverGroup
+
+## SYNOPSIS
+Removes an Instance Failover Group.
+
+## SYNTAX
+
+### RemoveIFGDefault (Default)
+```
+Remove-AzSqlDatabaseInstanceFailoverGroup [-ResourceGroupName] <String> [-Location] <String>
+ [-Name] <String> [-Force] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### Remove a Instance Failover Group from Resource Id
+```
+Remove-AzSqlDatabaseInstanceFailoverGroup [-Location] <String> [-ResourceId] <String> [-Force]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### Remove a Instance Failover Group from AzureSqlInstanceFailoverGroupModel instance definition
+```
+Remove-AzSqlDatabaseInstanceFailoverGroup -InputObject <AzureSqlInstanceFailoverGroupModel> [-Force]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+This command removes the Instance Failover Group with the specified name, leaving all databases intact. The listener endpoint will be unregistered from DNS.
+
+The Instance Failover Group's primary region should be used to execute the command.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Get-AzSqlDatabaseInstanceFailoverGroup -ResourceGroupName rg -Location location -Name fg | Remove-AzSqlDatabaseInstanceFailoverGroup
```

Comment:
@ziwa-msft does this cmdlet write anything to the output stream? The general pattern is that `Remove-*` cmdlets return nothing unless the user provides the [`-PassThru` parameter](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/design-guidelines/cmdlet-best-practices.md#returning-no-output), in which case the value of `true` is written to the output stream.

I'm not sure if this is the case or even possible with how the Sql cmdlets are structured, so just let me know.

Reply:
I don't think it currently returns anything (will double check). This is the expected pattern right, if I did not mistake your explanation above?

Reply:
@ziwa-msft the only thing that we would need to add here is the `-PassThru` parameter to allow the cmdlet to return output if the user decides they want something returned

Reply:
Added the PassThru parameter as requested.